graphics/palette.h

```diff
@@ -153,11 +252,10 @@ class PaletteLookup {
 	 *
 	 * @return the created map, or nullptr if one isn't needed.
 	 */
-	uint32 *createMap(const byte *srcPalette, uint len, bool useNaiveAlg = false);
+	uint32 *createMap(const Palette &srcPalette, bool useNaiveAlg = false);
```

Comment:
Could you keep the old version of this for convenience?

Reply:
I'm currently using it for an external WIP engine: https://github.com/ccawley2011/scummvm/blob/noot/engines/noot/widgets.cpp#L349
Having the ability to directly pass byte pointers means that it's not necessary to make a copy of the palette in order to create the map.